tests/cloudformation/graph/checks/resources/MSKClusterLogging/expected.yaml

```diff
@@ -1,9 +1,9 @@
 pass:
-  - "AWS::MSK::Cluster.ClusterCloudWatchLogsGood"
-  - "AWS::MSK::Cluster.ClusterFirehoseGood"
-  - "AWS::MSK::Cluster.Clusters3Good"
-  - "AWS::MSK::Cluster.ClusterAllGood"
+  - "ClusterCloudWatchLogsGood"
+  - "ClusterFirehoseGood"
+  - "Clusters3Good"
+  - "ClusterAllGood"
 fail:
-  - "AWS::MSK::Cluster.ClusterBadNoLoggingInfo"
-  - "AWS::MSK::Cluster.ClusterBadNoLoggingDisabled"
+  - "ClusterBadNoLoggingInfo"
+  - "ClusterBadNoLoggingDisabled"
```

Comment:
@Saarett Resource type should be part of the resource ID

Reply:
@nimrodkor Resource type is still part of the vertex id, I've changed the vertex name from _resource_type.resource_name_ to _resource_name_ so it affects the expected results of this test